diff --git a/cmake/urdfdom_headers-config.cmake.in b/cmake/urdfdom_headers-config.cmake.in
index 74a0db6..632a398 100644
--- a/cmake/urdfdom_headers-config.cmake.in
+++ b/cmake/urdfdom_headers-config.cmake.in
@@ -3,4 +3,4 @@ if (@PACKAGE_NAME@_CONFIG_INCLUDED)
 endif()
 set(@PACKAGE_NAME@_CONFIG_INCLUDED TRUE)
 
-set(@PACKAGE_NAME@_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include")
+set(@PACKAGE_NAME@_INCLUDE_DIRS "include")
